src/sentry/api/serializers/models/group.py

```diff
@@ -195,8 +197,12 @@ def serialize(self, obj, attrs, user):
             status_details['autoResolved'] = True
         if status == GroupStatus.RESOLVED:
             status_label = 'resolved'
-            if attrs['pending_resolution']:
-                status_details['inRelease'] = attrs['pending_resolution']
+            if attrs['resolution']:
+                res_type, res_version = attrs['resolution']
+                if res_type in (GroupResolution.Type.in_next_release, None):
```

Comment:
So, worth noting, because I think @tkaemming also assumed this pattern in a previous PR.

Since the column is nullable, we NULL is literally free for Postgres in terms of size, we opted to treat the default value itself as NULL and never store a 0, only a 1.

So in this case, `GroupResolution.Type.in_next_release` itself would be `None` instead of 0. This also avoids the case of needing to check for both 0 or None and saving on disk space (which isn't really that much of a concern here since the table is significantly smaller)

Just two cents if you like the idea or not.